eng/linkerTests/project.csproj.template

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
```

Comment:
Why is this hard-coded?

Reply:
This will be changed and that is what I meant with my first item in the list of checkboxes that still needs to happen before this gets merged. The way it will work, is that the LinkerTest.csproj will now provide an item called `TestRuntimeIdentifiers` and we will create one publish project per RID. The idea is also that when you are running locally, we will only test the current RID platform, but if you are running for helix, we will publish for all of those different items and run them in different queues.